Chart/assets/js/api.js

```diff
@@ -0,0 +1,40 @@
+const currency = JSON.parse(localStorage.getItem('currency'));
+let Exchange;
+
+
+if(currency){
+        console.log(currency);
+}
+else{
+axios.get('https://marketdata.tradermade.com/api/v1/live_currencies_list?api_key=IhNDplgSDJWAxAXFyIKH')
+        .then(response => {
+        localStorage.setItem('currency',JSON.stringify(response.data));  
+        })
+        .catch(error => {
```

Comment:
please handle error in a user-friendly way and show message of the error to be descriptive 
also you can use toast or div with this descriptive message for a better ux